api/src/apollo/Team/mutations.ts

```diff
@@ -0,0 +1,58 @@
+import { idArg, inputObjectType, list, mutationField, nonNull } from 'nexus';
+
+export const TeamCreateInputType = inputObjectType({
+  name: 'TeamCreateInputType',
+  definition(t) {
+    t.nonNull.id('id');
+    t.string('position');
+    t.string('team');
+    t.nonNull.id('userID');
+    t.nonNull.id('orgID');
+    t.int('priority');
+  },
+});
+
+export const createTeam = mutationField('createTeam', {
+  type: 'Int',
```

Comment:
Please add a spefic comment in this case as it's unusual case then normal create mutations where we aren't returning created object